src/caffe/layers/batch_norm_layer.cpp

```diff
@@ -232,5 +232,4 @@ STUB_GPU(BatchNormLayer);
 #endif
 
 INSTANTIATE_CLASS(BatchNormLayer);
-REGISTER_LAYER_CLASS(BatchNorm);
```

Comment:
Ah, because of the change in registration to switch between CAFFE and CUDNN engines, right?